### DISCUSSION TITLE
Datagrid - text align

### DISCUSSION THREAD

**Discussion by PoojMenon** [twilio-labs/paste#2019] [Q&A] [answered]
Hi, 

Trying to use the Datagrid in a Codesandbox and its throwing an error on the textAlign property.
![image](https://user-images.githubusercontent.com/72031986/140951377-3d7e34c5-1b4a-4430-9df8-ce26868dc468.png)

I can see on the doc site that the prop is available for `DataGridHeader` but not for `DataGridCell`. Trying to achieve the same as for the table [here](https://paste.twilio.design/components/table/#column-alignment).

**Comment by nkrantz** [comment, marked as answer]
Hey @PoojMenon, thanks for catching this. It was an oversight, and `DataGridCell` should accept the `textAlign` prop. I'm going to get a fix in ASAP, I'll post the PR link here when it's up. In the meantime, if you're willing to live with some red squiggly lines, adding the prop should still work.

**Reply by PoojMenon** [reply]
Thanks for the quick response @nkrantz

**Reply by nkrantz** [reply]
PR: https://github.com/twilio-labs/paste/pull/2025

**Reply by nkrantz** [reply]
PR is in! Let us know if you are still experiencing the issue. Thanks!

**Reply by PoojMenon** [reply]
That seems to be working - thanks very much!!
